proto/sequencerblockapis/astria/sequencerblock/v1alpha1/optimistic_block.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package astria.auctioneer.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+import "astria/execution/v1alpha2/execution.proto";
+import "astria/sequencerblock/v1alpha1/block.proto";
+import "astria/primitive/v1/types.proto";
+
+// The "BaseBlock" is the information needed to simulate bundles on top of
+// a Sequencer block which may not have been committed yet.
+message BaseBlock {
+    // This is the block hash for the proposed block.
+    bytes sequencer_block_hash = 1;
```

Comment:
why is this needed? this type is only passed into geth, is it just putting it in the `Bundle`s?

Reply:
I think this should probably be returned from the optimistic execution result stream, see above comment